style.css

```diff
@@ -0,0 +1,207 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@200&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  text-decoration: none;
+  font-family: 'Poppins', sans-serif;
+}
+
+.text-color {
+  color: white;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 24px;
+}
+
+.header-container {
+  display: flex;
+  justify-content: space-between;
+  margin-right: 10%;
+  margin-left: 10%;
+  align-items: center;
+  padding: 6px;
+}
+
+.main-header {
+  background-color: #3c3a39;
+  width: 100vw;
+}
+
+.menubar {
+  height: 20px;
+}
+
+.section-1 {
+  background-color: #1c1a19;
+  height: 100vh;
+  width: 100vw;
+}
+
+.content-0 {
+  width: 100vw;
+  height: 100vh;
+  padding-top: 37%;
+  background-image: url(./socialmedia/design.png);
+  background-repeat: no-repeat;
+  background-position: 100% 100%;
+}
+
+.heading-1-container {
+  margin-bottom: 20px;
+}
+
+.margin-24 {
+  margin: 0 24px;
+}
+
+.heading-1 {
+  color: white;
+  font-family: 'Crete Round', sans-serif;
```

Comment:
Good job for adding the good font-family 👍 
Unfortunately, you forgot to [import it](https://fonts.google.com/specimen/Crete+Round) and to change the color, please correct the issue to stick Figma template. 


Your design
<img width="422" alt="image" src="https://user-images.githubusercontent.com/7243132/170366481-91a908f6-1f4e-4b6b-b89a-f5e57dda3f65.png">

Figma
<img width="303" alt="image" src="https://user-images.githubusercontent.com/7243132/170366522-cc019394-d861-4452-8a15-d4a9ee9ad0bc.png">